jetty-core/jetty-compression/jetty-compression-brotli/src/main/config/modules/compression-brotli.mod

```diff
@@ -10,13 +10,13 @@ compression
 experimental
 
 [depend]
-compression-common
 compression
 
 [files]
 maven://com.aayushatharva.brotli4j/brotli4j/${brotli4j.version}|lib/compression/brotli4j-${brotli4j.version}.jar
 
 [lib]
+lib/compression/jetty-compression-common-${jetty.version}.jar
```

Comment:
No, this goes in the `compression.mod` module.
Same for all the others.

Reply:
You asked me to remove the parent dependency in https://github.com/jetty/jetty.project/pull/12531#discussion_r1846791405

This is the result.

Before I make any more changes to the start modules here.
Lets plan ahead, what are the modules going to be called for jetty-client?
How are we going to make the differences clear in all of the modules here?

At this point in time on this branch we have ...

* `compression` - the server handler
* `compression-gzip` - support for gzip in server handler
* `compression-brotli` - support for brotli in server handler
* `compression-zstandard` - support for zstandard in server handler

When we introduce client compression support, what will those new names be?

* `client-compression` - generic client compression support.
* `client-compression-gzip` - decompressing gzip content-encoding responses on client
* `client-compression-brotli` - decompressing brotli content-encoding responses on client
* `client-compression-zstandard` - decompressing zstandard content-encoding responses on client

Now one step further, if someone wants to build their own compression support (lets say it is for bzip2), and they want to integrate it into the start module system, do we have a sufficiently clean and easy to follow module naming above to make things easy for them?